x/service/types/genesis_test.go

```diff
@@ -6,9 +6,20 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/pokt-network/poktroll/x/service/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
 func TestGenesisState_Validate(t *testing.T) {
+	srv1 := &sharedtypes.Service{
+		Id:   "srv1",
```

Comment:
NIT (here and below): Can you name it `srvId1` and `srvName1` (or something like that) so they're different.